demos/mcp_server_memory/memory_service/core_memory_service.py

```
@@ -0,0 +1,360 @@
+
+    except Exception as e:
+        return format_response(
+            success=False,
+            message=f"Error initializing memory system: {str(e)}"
+        )
+
+
+def store_memory(
+        content: str,
+        topic: str,
+        tags: List[str] = []
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)